tools/nodejs/lib/duckdb.js

```diff
@@ -252,7 +252,7 @@ Connection.prototype.register = function (name, return_type, fun) {
             console.log(desc.ret);
             msg = error;
             if (typeof error == 'object' && 'message' in error) {
-                msg = error.message
+                msg = error.Message()
```

Comment:
I don't think you've changed the shape of the exposed error at all?

Reply:
Ah you're right, I missed that